ember-client/src/main/scala/org/http4s/ember/client/EmberClientBuilder.scala

```diff
@@ -166,20 +172,6 @@ final class EmberClientBuilder[F[_]: Concurrent: Timer: ContextShift] private (
                   userAgent
                 )
             )
-            .map(response =>
```

Comment:
`ClientHelpers` already has this logic, correct me if this actually necessary twice

Reply:
That looks correct.